packages/datagateway-search/src/search/searchButton.component.tsx

```diff
@@ -91,26 +103,31 @@ class SearchButton extends React.Component<SearchButtonCombinedProps> {
   public handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
     if (this.props.dataset === true) {
       let datasetParams = this.urlParamsBuilder('Dataset');
-      this.sendRequest(datasetParams);
+      this.luceneRequest(datasetParams);
     }
     if (this.props.datafile === true) {
       let datafileParams = this.urlParamsBuilder('Datafile');
-      this.sendRequest(datafileParams);
+      this.luceneRequest(datafileParams);
     }
     if (this.props.investigation === true) {
       let investigationParams = this.urlParamsBuilder('Investigation');
-      this.sendRequest(investigationParams);
+      this.luceneRequest(investigationParams);
     }
   };
 
-  public async sendRequest(queryParams: LuceneParameters): Promise<void> {
+  public async luceneRequest(queryParams: LuceneParameters): Promise<void> {
     const response = await axios.get(
       'https://scigateway-preprod.esc.rl.ac.uk:8181/icat/lucene/data',
       { params: queryParams }
     );
-    console.log(response);
+
+    const ids = response.data.map((x: { id: number; score: number }) => x.id);
+    console.log(ids);
   }
 
+  //  Louise does this already, find her work
```

Comment:
Again, I presume this is a WIP comment - can you either remove or clearly label TODO so this is removed later on?